src/elements/Each.tsx

```diff
@@ -23,7 +23,7 @@ export default function Each(props: HenshuEachProps) {
     const [items, setItems] = useState(get());
     const [selection, setSelection] = useState<Selection|null>(null);
     const [selectionBox, setSelectionBox] = useState<Element|null>(null);
-    const timeout = useRef();
+    const timeout = useRef<number | undefined>();
```

Comment:
@arminakvn is this the only real change in this PR?   Are the changes in dist just the compiled version?  If so I think we may want to git ignore the dist folder from source control so the changes on generated code are not stored.

Reply:
Thank you so much for your comments! Yes, that is the only code change-- the upgraded packages in the package.json file are what is needed for updating the henshu-app package (which is going to be coming after this goes through) -- regarding the ignoring the dist folder, that is actually something I've seen we do all different kinds of ways and I am not sure what is best practice? I usually just keep it the way I clone down the repositories, and in this case the dist is not ignored and I thought maybe that's a practice for npm libraries or something (in fact I think most of the packages in lib folder are like this). What is your suggestion going forward?